### DISCUSSION TITLE
Compare datahike to more databases

### DISCUSSION THREAD

**Discussion by jsmassa** [replikativ/datahike#418] [Ideas]
### Describe the feature you would like to request

To help other developer to decide for a database, so far, we describe differences to 
- DataScript 
- Datomic

This topic should be extended to other Datalog databases like
- XTDB
- Asami
- Datalevin

### Describe the solution you would like

Write a new document addressing all of those databases.

### Describe alternatives you've considered

Extending the section in the README.md would be also possible, but the section would become quite large.

**Comment by kordano** [comment]
@whilo How would you like to approach this?

**Comment by whilo** [comment]
You can also just point to https://clojurelog.github.io/ and skip specific opinions.
